### DISCUSSION TITLE
How to Resolve this Error of ths repo?

### DISCUSSION THREAD

**Discussion by shary998** [florisboard/florisboard#1929] [Q&A]
Cause: org/gradle/internal/impldep/com/google/common/base/CharMatcber$NamedFastMatcber (wrong name: org/gradle/internal/impldep/com/google/common/base/CharMatcher$NamedFastMatcher)

**Comment by patrickgold** [comment]
> CharMatcber

This name is definitely misspelled, however I do not explicitly know where this class is used, so I don't know where this error comes from tbh.
